datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/py3.py

```diff
@@ -21,11 +21,13 @@
 
 @click.command(
     context_settings=CONTEXT_SETTINGS,
-    short_help="Verify if a custom check or integration will run on python 3"
+    short_help="Verify if a custom check or integration can run on python 3"
 )
 @click.argument('check')
 def py3(check):
-    """Verify if a custom check or integration will run on python 3"""
+    """Verify if a custom check or integration can run on python 3. CHECK
+    must be a valid path to a Python module or package folder.
```

Comment:
Small not, this only has to be the name of an integration, not necessarily the path since we will construct the path based on the name right?